src/commons/navigationBar/subcomponents/AcademyNavigationBar.tsx

```diff
@@ -152,4 +51,86 @@ export const icons: IconName[] = [
   IconNames.CALCULATOR
 ];
 
+export const assessmentTypesToNavlinkInfo = ({
+  assessmentTypes,
+  courseId,
+  isEnrolledInACourse
+}: {
+  assessmentTypes?: string[];
+  courseId?: number;
+  isEnrolledInACourse: boolean;
+}): NavbarEntryInfo[] =>
+  (assessmentTypes || []).map((assessmentType, idx) => ({
+    to: `/courses/${courseId}/${assessmentTypeLink(assessmentType)}`,
```

Comment:
What happens if `courseId` is `undefined`?

Reply:
This component is only rendered when `role` is defined, in which case `courseId` should also be defined. To make it more robust, I added an additional early return to not render this component when `courseId` or `role` is undefined